apps/server/src/modules/tool/common/uc/tool-permissions-helper.spec.ts

```diff
@@ -46,6 +46,10 @@ describe('ToolPermissionHelper', () => {
 					provide: AuthorizationService,
 					useValue: createMock<AuthorizationService>(),
 				},
+				{
```

Comment:
Kommt die Änderung von euch, oder vom merge?

Reply:
from Merg is reverted.